project/plugins.sbt

```diff
@@ -1,13 +1,13 @@
 val scalaJSVersion =
-  Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.23")
+  Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.26")
```

Comment:
Is there a need to bump SJS version, as it forces the new version down to dependencies, too?